ext/fetch/23_response.js

```diff
@@ -209,6 +210,18 @@
       return response;
     }
 
+    /**
+     * @param {any} value
+     * @param {ResponseInit} init
+     * @returns {Response}
+     */
+    static json(value, init = {}) {
+      const body = JSONStringify(value);
+      const response = new Response(body, init);
+      response.headers.set("Content-Type", "application/json");
```

Comment:
The `Response` constructor when passed a string, `ArrayBuffer`, `FormData`, etc. will only set the `Content-Type` header if one wasn't passed in the `init` object. `Response.json()` should probably do the same.

Reply:
Handled by 2142d7b61